src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/mobile-sdk-api-guide.mdx

```diff
@@ -0,0 +1,363 @@
+      </td>
+    </tr>
+
+            <tr>
+      <td>
+        Configure cross-app tracing to link transactions between your mobile and APM-monitored applications.
+      </td>
+
+      <td>
+        [Configure cross-app tracing](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/cross-app-tracing)
+      </td>
+    </tr>
+
+                <tr>
+      <td>
+        Run a test crash report.
+      </td>
+
+      <td>
+        [Test crash reporting](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/test-crash-reporting)
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Android usage guide [#android-sdk-usage]
+
+### Automatically instrumented methods [#method-list]
+
+Automatic instrumentation is one of the agent's more convenient features. However, if it interferes with the method being traced, add `@SkipTrace` to the method, and the agent will skip it during compile time instrumentation.
+
+    <table>
+      <thead>
+        <tr>
+          <th width={200}>
+            **Classes**
+          </th>
+
+          <th>
+            **Methods**
+          </th>
+        </tr>
+      </thead>
+
+      <tbody/>
+
+      <tbody>
+        <tr>
+          <td>
+            `Activity`
+          </td>
+
+          <td>
+            * `onCreate`
+            * `onCreateView`
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `AsyncTask`
+          </td>
+
+          <td>
+            * `execute`
+            * `executeOnExecutor`
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `BitmapFactory`
+          </td>
+
+          <td>
+            * `decodeFile`
+            * `decodeResourceStream`
+            * `decodeResource`
+            * `decodeByteArray`
+            * `decodeStream`
+            * `decodeFileDescriptor`
+            * `decodeResourceStream`
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `SQLiteDatabase`
+          </td>
+
+          <td>
+            * `query`
+            * `queryWithFactory`
+            * `rawQuery`
+            * `rawQueryWithFactory`
+            * `insert`
+            * `insertOrThrow`
+            * `insertWithOnConflict`
+            * `replace`
+            * `replaceOrThrow`
+            * `delete`
+            * `update`
+            * `updateWithOnConflict`
+            * `execSql`
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `GSON`
+          </td>
+
+          <td>
+            * `toJson`
+            * `fromJson`
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `JSONObject`
+          </td>
+
+          <td>
+            * `<init>`
+            * `toString`
+          </td>
+        </tr>
+
+        <tr>
+          <td>
+            `JSONArray`
+          </td>
+
+          <td>
+            * `<init>`
+            * `toString`
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+## iOS usage guidelines [#ios-sdk-usage]
```

Comment:
Note to self - build these sections out with SMEs, or rename the section in the meantime and create a followup ticket